### ISSUE TITLE
Python 3.13 - ModuleNotFoundError: No module named 'pipes'

### ISSUE THREAD

**Issue by rpetit3** [rpetit3/fastq-dl#33]
I will need to make some fixes on the bioconda recipe. But in case you get the following erro

```
ModuleNotFoundError: No module named 'pipes'
```

The solution is to downgrade your Python version to 3.12 or earlier.

**Comment by mbhall88** [comment]
Weird. We don't have a dependency on a module called pipes do we?

**Comment by rpetit3** [comment]
Yep we do (and its not in the pyproject .toml which is another issue itself haha) ,  https://github.com/rpetit3/fastq-dl/blob/master/fastq_dl/utils.py#L11


I've submitted a PR here https://github.com/conda-forge/executor-feedstock/pull/8 , based on this PR https://github.com/xolox/python-executor/pull/22

Executor is a package I've used for years for simplifying the usage of subprocess. We can reconsider its usage going forward, but if the PR fixes it (substitutes `pipes` for `shlex`) then I think we just leave it as is.

**Comment by mbhall88** [comment]
yeah I was going to say `subprocess`, which is in the standard library is pretty easy to work with so I think we could very easily transition to that.

**Comment by rpetit3** [comment]
Heads up planning to work on dropping the wrapper and just going with subprocess directly

**Comment by apcamargo** [comment]
I've been using [`sh`](https://sh.readthedocs.io/) for a while now to simplify calling external programs. I guess it could replace Executor.

**Comment by rpetit3** [comment]
Hi everyone, 

Thank you for the suggestions. For the time being, I have released an update that pins  Python to <3.13. There are other issues unrelated to this, but related to 3.13 deprecations. 

I will tackle v3.13 compatibility in a future release.

Thanks again,
Robert
